src/OpenColorIO/CMakeLists.txt

```diff
@@ -218,11 +218,10 @@ if(WIN32)
 endif()
 
 set_target_properties(OpenColorIO PROPERTIES
-	COMPILE_FLAGS "${PLATFORM_COMPILE_FLAGS}")
-
-set_target_properties(OpenColorIO PROPERTIES
-    VERSION ${OpenColorIO_VERSION}
-    SOVERSION ${SOVERSION})
+    OUTPUT_NAME   OpenColorIO${OCIO_LIBNAME_SUFFIX}
```

Comment:
Something I learn. 

[Not mandatory] As you are reworking the `OUTPUT_NAME`, you could:
1. Change `LIBNAME_SUFFIX` to be ` _${OpenColorIO_VERSION_MAJOR}_${OpenColorIO_VERSION_MINOR}`
i.e. prepended an underscore;
2. Append `${LIBNAME_SUFFIX}` (i.e. which will be `_2_0` for Windows and empty for all other platforms)
i.e. `OUTPUT_NAME   OpenColorIO${OCIO_LIBNAME_SUFFIX}${LINAM_SUFFIX}`;
3. Copy & paste lines 206 to 210 to replace the comment at line 133;
4. Remove lines 204 to 215.

I think that it nicely completes the CMakeLists.txt cleanup around the right output name for the library.

Reply:
Are these things you want me to do in this PR, or is this just a mental note for things to improve further later?

Reply:
In the PR if possible.

Reply:
Pushed an update that implements these suggestions. Also fixed version.rc.in.